src/main/java/com/faforever/client/remote/domain/VictoryCondition.java

```diff
@@ -1,44 +1,16 @@
 package com.faforever.client.remote.domain;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
 import lombok.extern.slf4j.Slf4j;
 
-import java.util.HashMap;
-import java.util.Map;
-
 @Slf4j
+@Getter
 public enum VictoryCondition {
   // Order is crucial
-  DEMORALIZATION(0),
-  DOMINATION(1),
-  ERADICATION(2),
-  SANDBOX(3),
-  UNKNOWN("unknown");
-
-  private static final Map<Object, VictoryCondition> fromNumber;
-
-  static {
-    fromNumber = new HashMap<>();
-    for (VictoryCondition victoryCondition : values()) {
-      fromNumber.put(victoryCondition.value, victoryCondition);
-    }
-  }
-
-  private final Object value;
-
-  VictoryCondition(Object value) {
-    this.value = value;
-  }
-
-  public static VictoryCondition fromNumber(Object number) {
-    VictoryCondition victoryCondition = fromNumber.get(number);
-    if (victoryCondition == null) {
-      log.warn("Unknown victory condition: {}", number);
-      return UNKNOWN;
-    }
-    return victoryCondition;
-  }
-
-  public Object getValue() {
-    return value;
-  }
+  @JsonProperty("0") DEMORALIZATION,
+  @JsonProperty("1") DOMINATION,
+  @JsonProperty("2") ERADICATION,
+  @JsonProperty("3") SANDBOX,
```

Comment:
I tested and it works but I also have a better way to do it now so will change it

Reply:
oh wait never mind the unknown ruins my other way

Reply:
Bah nevermind I realize it doesn't work because the victory condition is read in java commons and that has a different type. Really the LocalReplayInfo probably should be in the commons library instead of here I think along with the VictoryCondition definition